src/pynws/nws.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         userid: str,
         latlon: Optional[Tuple[float, float]] = None,
         station: Optional[str] = None,
+        forecast_units: Optional[str] = None,
```

Comment:
I think we should be consistent between classes on input here.  I like requiring a str (bonus points for figuring out the typing to be either exactly "us" or "si") with the default being "us".

Reply:
That sounds like a `StrEnum`

Reply:
I updated the code to use an `StrEnum`. Let me know how this looks for you!